addons/vectors/fnc_matrixProduct3D.sqf

```diff
@@ -26,8 +26,4 @@ Author:
 
 params [["_matrixA", [], [[]], 3], ["_matrixB", [], [[]], 3]];
 
-_matrixB = [_matrixB] call CBA_fnc_matrixTranspose;
```

Comment:
where did the transpose go?

Reply:
No need for transpose anymore. It was just a workaround for an easier access to column vectors as 1D arrays, since **vectorDotProduct** only supports 1D arrays.